napari/layers/image/tests/test_pyramid.py

```diff
@@ -41,7 +41,7 @@ def test_error_pyramid():
     shapes = [(40, 20), (20, 10), (10, 5)]
     np.random.seed(0)
     data = [np.random.random(s) for s in shapes]
-    with pytest.raises(ValueError):
+    with pytest.raises(Exception):
```

Comment:
Why a bare exception? What is actually raised?

Reply:
Uninformative errors are raised - here we're basically pretending a pyramid data set is not a pyramid, but we removed our guess_pyramid from being called as it was too expensive. I'm not quite sure how to improve

Reply:
What is the *class* of error? We should check for that class rather than "Exception", as checking for Exception can mask unexpected errors.

Reply:
ok can improve - this might change by the end of the PR - lol